app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list:
+    list_of_person_instances = [
+        Person(name=person["name"], age=person["age"])
+        for person in people
+    ]
+
+    for person in people:
+        if "husband" in person and person["husband"] is not None:
```

Comment:
you can compare not only with None, just check if person["husband"] and it will be the same check.